src/routes.ts

```diff
@@ -34,8 +39,32 @@ const routes: Routes = [
     module: ReportWorkspaceModule,
   },
   {
-    path: '/error-suppressions',
-    module: ErrorSuppressionsModule,
+    path: '/admin',
+    module: AdminModule,
+    children: [
+      {
+        path: '/qa-certification',
```

Comment:
```suggestion
        path: '/qa-maintenance',
```

Reply:
I'll change the patch from /qa-certification to /qa-maintenance. I missed the naming on that from what's on the ticket.

I can change the files/modules to `qa-test`. I don't think it should be `qa-test-summary` since that data is joined with qa_supp_data and a lot of other tables as well (hence why the generic 'data' added to the name)

Reply:
Based on our discussion i'll change to qa-test-summary for the endpoint